travis.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Please move this file to more appropriate location. `.ci/travis.sh`?

Reply:
That'll be better. I'll move it.